test/parallel/test-crypto-key-objects.js

```diff
@@ -33,6 +34,26 @@ const privatePem = fixtures.readSync('test_rsa_privkey.pem', 'ascii');
   });
 }
 
+{
+  // Attempting to instantiate KeyObject with `new` should throw
```

Comment:
This comment is misleading. You are right, attempting to instantiate using `new` should throw, but it does not always (you can easily "fool" the code to create an instance). (Also, missing `.` at the end of the comment.)

Reply:
With the `handle instanceof KeyObjectHandle` check in place, how could i fool it?

Reply:
I did not try this, but I am pretty sure one can extract the `KeyObjectHandle` of an existing `KeyObject` using `Object.getOwnPropertySymbols`. And sure, I hope nobody will do that, I am just saying that the comment is not entirely correct, especially since you are testing for two different error messages below.

Reply:
hmmm, so just change the comment in tests?

Reply:
In theory, we *could* prevent people from calling the constructor entirely, but that would add complexity to the code and slightly impact performance... That is one of the reasons I don't like exposing these classes. Yeah, maybe just change the comment.

Reply:
> we _could_ prevent people from calling the constructor entirely

I would be interested in the how on this. Unrelated to this PR tho

Reply:
For example, we could use a `Symbol` that needs to be passed to the constructor and that is assigned to a `const` inside of the internal module, making it inaccessible from userland.

Reply:
@tniessen e.g. like this? https://github.com/nodejs/node/compare/master...panva:keyobject-no-instance would this be accepted as a solution?